conan.cmake

```diff
@@ -83,6 +84,22 @@ function(conan_cmake_settings result)
 endfunction()
 
 
+function(conan_cmake_detect_gnu_libcxx result)
+    get_directory_property(defines DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS)
+    foreach(define ${defines})
+        if(define STREQUAL "_GLIBCXX_USE_CXX11_ABI=0")
+            set(${result} libstdc++ PARENT_SCOPE)
+            return()
+        endif()
+    endforeach()
+    if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "5.1")
```

Comment:
The difficult part of this, and the reason why libcxx is explicitely modeled as a setting, parallel to the version one, is that this is not always true: A typical installation of gcc 5.3/5.4 in a (mainstream) Ubuntu 14, will not install libstdc++11, but libstdc++ (due to system/installed apps incompatibilities).

I guess that it is enough defining ``cmake -D_GLIBCXX_USE_CXX11_ABI=0`` in the command line, is it? Not very sure about what is the behavior of ``get_directory_property`` for the defines